docs/mine-hnt/validators/validator-score.mdx

```diff
@@ -1,39 +1,97 @@
+**25% of the Validators in the consensus group are ejected:**
+* This is a probabilistic determination based on the current penalty scores of the Validators in the current consensus group.
+* Validators with _higher_ penalty score will have a _higher_ chance of being ejected
+* Validators with _lower_ penalty score will have a _lower_ chance of being ejected
 
-The negative impact of a higher score (early removal or not being selected) is weighted based on comparison with scores of other validator nodes in the consensus group.
+
+**The ejected 25% is replaced by available Validators:**  
+* This is also probabilistic, based on the current penalty scores of the Validators available to be included in the consensus group.
+* Validators with _higher_ penalty score will have a _lower_ chance of being elected
+* Validators with _lower_ penalty score will have a _higher_ chance of being elected
+
+It's important to understand that elections are still based on probability. Merely having a high penalty will not ensure a Validator is removed from a consensus group. By contrast, having a low penalty score does not ensure that a Validator will _remain_ in a consensus group. 
+
+Upon joining the Network, Validator nodes start with a base penalty of `0.0`. 
+
+## Penalty Types
+
+Now that there's a high-level description of how penalties affect the election process, let's define how penalties are assessed:
+
+### Consensus Group Participation
+
+To ensure that consensus groups members are continually churned, and that no individual Validator participates for too long, penalties are added. This is not a performance based penalty and is added to _all_ Validators that participate in a consensus group.
+
+A penalty of `1.0` is added at the end of each consensus group for each Validator that participated in that group.  If the Validator is not ejected from the consensus group, it will receive another `1.0` penalty at the end of the 2nd consensus group, now having a score of `2.0`.  
+
+This penalty of `1.0` is added even if the Validator is ejected from the consensus group as it is a penalty added for participating in the ending consensus group.
+
+
+### Performance Based
+
+The details of the HoneyBadgerBFT protocol used is out of scope for this document, but this can be read more about [here](https://eprint.iacr.org/2016/199.pdf).  What's important is that there are a few different types of failures that will create performance penalties that can be applied to a Validator while in consensus:
+
+**Seen Penalties**
+* This penalty can occur if a majority of other Validators in the consensus group report your Validator as not being "seen"
+* This is equivalent to the Validator being offline, not available, or not responding to the group.
+* This is a penalty of `0.33` for each occurrence
```

Comment:
I think this is worded a bit awkwardly. How about something like this (would apply to this and BBA section)

* This penalty is applied for each occurrence, which can happen multiple times during an consensus group.
* The amount applied per occurrence is a chain variable, currently set at 0.33. The variable is viewable by running `miner ledger variables election_seen_penalty`